cmd/ctrl.go

```diff
@@ -114,14 +122,24 @@ var ctrlCmd = &cobra.Command{
 			}
 			return jsonPrint(cfs)
 		case CtrlQueryCaptures:
-			_, captures, err := kv.GetCaptures(context.Background(), cli)
+			_, raw, err := kv.GetCaptures(context.Background(), cli)
 			if err != nil {
 				return err
 			}
+			mockOwnerMgr := roles.NewOwnerManager(cli, "mock-id", kv.CaptureOwnerKey)
+			ownerID, err := mockOwnerMgr.GetOwnerID(context.Background())
```

Comment:
Maybe just extract a standalone function to get owner ID? It seems the only inputs we need are just `key` and `ctx`.